test/roundtrip/apply-multi-memories.txt

```diff
@@ -0,0 +1,25 @@
+;;; TOOL: run-roundtrip
```

Comment:
How about calling this `apply-memory-names.txt` to match the existing `apply-global-names.txt`